web/controller/components/controlBulletin/ControlBulletinFormStep1.js

```diff
@@ -3,41 +3,99 @@ import React from "react";
 import Stack from "@mui/material/Stack";
 import { Select, TextInput } from "@dataesr/react-dsfr";
 import { COUNTRIES } from "../../utils/country";
+import { DEPARTMENTS } from "../../utils/departments";
+import { useApi } from "common/utils/api";
+import { CONTROL_LOCATION_QUERY } from "common/utils/apiQueries";
+import { DsfrAutocomplete } from "../utils/DsfrAutocomplete";
 
 export function ControlBulletinFormStep1({
   handleEditControlBulletin,
   controlBulletin,
   showErrors
 }) {
+  const [controlLocationCommunes, setControlLocationCommunes] = React.useState(
+    []
+  );
+  const [controlLocationLabels, setControlLocationLabels] = React.useState([]);
+  const [departmentLocations, setDepartmentLocations] = React.useState([]);
```

Comment:
:orange_circle: Je me demande toujours s'il vaut mieux procéder ainsi : plusieurs states ou n'avoir qu'un state et avoir des champs calculés par rapport à ce state.

> Can you compute it based on any other state or props in your component? If so, it isn’t state.

Source : https://legacy.reactjs.org/docs/thinking-in-react.html#step-3-identify-the-minimal-but-complete-representation-of-ui-state